openchain-blockchain-coinprism/azuredeploy.json

```diff
@@ -255,8 +241,8 @@
       "name": "[variables('vmName')]",
       "location": "[parameters('location')]",
       "dependsOn": [
-        "[concat('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]",
-        "[concat('Microsoft.Network/networkInterfaces/', variables('nicName'))]"
+        "[resourceId('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]",
```

Comment:
the storageAccount doesn't appear to ever be used, just remove it from the template.